nservicebus/messaging/routing.md

```diff
@@ -59,7 +59,7 @@ switch (Routing type?)
 
 Endpoint is a logical concept that relates to a program that uses NServiceBus to communicate with other similar programs. Each endpoint has a name e.g. `Sales` or `OrderProcessing`.
 
-During deployment each endpoint might be materialized in form of one or many instances. Each instance is an identical copy of binaries resulting from building the endpoint program code base. Usually each endpoint instance is placed in separate directory or separate machine. Each instance has its unique identity that consists of the name of the endpoint (e.g. `Sales`) and a discriminator. The discriminator can be provided either by the hosting infrastructure (e.g. Azure role ID) or by the user himself. In addition to these, an endpoint instance identity can include transport-specific information (e.g. machine name in MSMQ or schema name in SQL Server). The discriminator can be absent.
+During deployment each endpoint might be materialized in form of one or many instances. Each instance is an identical copy of binaries resulting from building the endpoint program code base. Usually each endpoint instance is placed in separate directory or separate machine.
```

Comment:
> Usually each endpoint instance is placed in separate directory or separate machine.

May be better as:

> Usually each endpoint instance is placed on a separate machine in order to scale out message processing.

I don't think it makes sense to talk about a separate directory - competing consumers on the same hardware shouldn't provide any benefit.